chemprop/data/datasets.py

```diff
@@ -347,6 +355,10 @@ def __getitem__(self, idx: int) -> list[Datum]:
     def smiles(self) -> list[list[str]]:
```

Comment:
We do this for the names property by default for ease of use when saving predictions. I have left `smiles` unchanged so that at the moment users can access either the individual or joint values depending on what is wanted.